examples/agentInteractionsDemo/index.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Agent-level Evaluations Page</title>
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
```

Comment:
Can you make sure all of these imports are still used?  Fine if they are, but if they are remnants from older versions, makes sense to pull them out.

Reply:
Just made sure they are all used. As jQuery and Popper.js are pre-requisites for Bootstrap and Bootstrap is used in the example app.